tests/unit/__snapshots__/ListTable.test.js.snap

```diff
@@ -69,7 +69,7 @@ exports[`when rendering a list-table directive renders correctly 1`] = `
   class="css-43thj7-TableContainer eb066o60"
 >
   <h5
-    class="css-s7wj1g-H5-commonHeading-bottomMargin e1top80012"
+    class="css-s7wj1g-H5-commonHeading-bottomMargin e1top80013"
```

Comment:
not sure why these 2 snapshot tests were failing. updated them bc they were blocking me from committing. it doesn't look related to any of my code changes.